app/views/stats/index.html.erb

```diff
@@ -74,25 +77,36 @@
-                <span class="words">(<%= ts("%{wordcount} words", wordcount: number_with_delimiter(work.word_count)) %>)</span>
+                <%= link_to work.title, work.item_path %>
+                <% if work.series? %>(Series)<% end %>
+                <% if (params[:view_type] != nil && params[:view_type] != "fandom") %><span class="fandom">(<%= work.fandom_string %>)</span><% end %>
+                <% if work.work? %><span class="words">
+                  (<%= t(".word_count_with_input", wordcount: number_with_delimiter(work.word_count)) %>)</span>
+                <% end %>
               </dt>
               <dd>
                 <dl class="stats">
-                  <% if (work_subscriber_count = Subscription.where(:subscribable_id => work.id, :subscribable_type => 'Work').count) > 0 %>
-                    <dt class="subscriptions"><%= ts("Subscriptions:") %></dt>
-                    <dd class="subscriptions"><%= number_with_delimiter(work_subscriber_count) %></dd>
+                  <% if work.series? %>
+                    <dt class="work_count"><%= t(".work_count") %>:</dt>
+                    <dd class="work_count"><%= number_with_delimiter(work.work_count) %> </dd>
                   <% end %>
-                  <dt class="hits"><%= ts("Hits:") %></dt>
-                  <dd class="hits"><%= number_with_delimiter(work.hits) %></dd>
-                  <% # dt ts("Downloads:") /dt dd work.downloads /dd %>
-                  <dt class="kudos"><%= ts("Kudos:") %></dt>
-                  <dd class="kudos"><%= number_with_delimiter(work.kudos.count) %></dd>
-                  <dt class="comments"><%= ts("Comment Threads:") %></dt>
-                  <dd class="comments"><%= number_with_delimiter(work.comment_thread_count) %></dd>
-                  <dt class="bookmarks"><%= ts("Bookmarks:") %></dt>
-                  <dd class="bookmarks"><%= number_with_delimiter(work.bookmarks.count) %></dd>
+                  <% if work.subscriptions_count > 0 %>
+                    <dt class="subscriptions"><%= t(".subscriptions") %>:</dt>
+                    <dd class="subscriptions">
+                      <%= number_with_delimiter(work.subscriptions_count) %>
+                      </dd>
+                  <% end %>
+                  <% if work.work? %>
+                    <dt class="hits"><%= t(".hits") %>:</dt>
+                    <dd class="hits"><%= number_with_delimiter(work.hits) %> </dd>
+                    <%# dt ts("Downloads:") /dt dd work.downloads /dd %>
+                    <dt class="kudos"><%= t(".kudos") %>:</dt>
+                    <dd class="kudos"><%= number_with_delimiter(work.kudos_count) %></dd>
+                    <dt class="comments"><%= t(".comment_threads") %>:</dt>
+                    <dd class="comments"><%= number_with_delimiter(work.comment_thread_count) %></dd>
+                  <% end %>
+                  <dt class="bookmark-count"><%= t(".bookmarks") %>:</dt>
```

Comment:
The class here should be `bookmarks` to correspond with what we use on works and series blurbs/meta. (We also as a general rule only use single-word class names, although there are a few exceptions, like third-party JavaScript.)

Reply:
That's odd. Rubocop shouldn't care about HTML class names, especially not one we use in many places across the site. You can just ignore it.